compiler/src/iree/compiler/Codegen/LLVMGPU/test/test_query_mma.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: iree-opt --split-input-file --iree-test-llvmgpu-query-mma %s | FileCheck %s
+
+#executable_target_rocm_hsaco_fb = #hal.executable.target<"rocm", "rocm-hsaco-fb",
+{abi = "hip", iree.gpu.target = #iree_gpu.target<arch = "gfx942", features = "",
+wgp = <compute =  fp64|fp32|fp16|int64|int32|int16|int8, storage =  b64|b32|b16|b8,
```

Comment:
We don't need an exhaustive list all the other wgp properties -- we can trim it down to something minimal like `compute = int32, storage = b32, ...`